lib/galaxy/datatypes/tabular.py

```diff
@@ -107,7 +113,17 @@ def displayable(self, dataset: DatasetInstance):
         except Exception:
             return False
 
-    def get_chunk(self, trans, dataset, offset=0, ck_size=None):
+    def get_chunk(self, trans, dataset: "DatasetInstance", offset: int = 0, ck_size: Optional[int] = None) -> str:
+        ck_data, last_read = self._read_chunk(trans, dataset, offset, ck_size)
+        return dumps(
+            {
+                "ck_data": util.unicodify(ck_data),
```

Comment:
The mode is "r" for get_fileobj, so this shouldn't be necessary

Reply:
Thanks, leaving it as is for now.